testutil/testkeyring/accounts_table.go

```diff
@@ -1,111 +1,117 @@
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMTgwODJnZzU3dHlwd3ZxM2N0YWNhZzljcWg5ejJqZjdkbDJzeDlhIiwiTW5lbW9uaWMiOiJsb3ZlIG1lbHQgZ3JpZCBzaGFsbG93IGZhY3VsdHkgdG9zcyBzdXJyb3VuZCBpbWl0YXRlIG5lY2sgY2F0IGJhc2ljIHNsaWdodCBqYWd1YXIgYWNyb3NzIGdpdmUgYWJzZW50IHJlZ2lvbiBncmF2aXR5IGVuZG9yc2UgYnJpZ2h0IHNpbXBsZSBvY3RvYmVyIHJldW5pb24gc3RvcnkifQ=="),
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMWxqd3Jsczg3MnB4MnEwNWFyZmdrazZnajQ0YXBxOHM1ajk5Z3Q1IiwiTW5lbW9uaWMiOiJzY291dCBtb2JpbGUgcmVjYWxsIG1lbWJlciByYWxseSB0aWx0IGNhcmQgaW50byB1bWJyZWxsYSBmZWVkIHZlcnNpb24gbm92ZWwgZ2lyYWZmZSBoYW1tZXIgem9uZSBrZWVuIG9yaWVudCBoYWxmIGRpc2FncmVlIGJlYWNoIHBvZXQgaW1wYWN0IGRpbm5lciBtYXNrIn0="),
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMTVzYTc4eTRkZ2tlc3oyN3k5bTZyNDByMDBzZTdtd2h5MjU2dGxoIiwiTW5lbW9uaWMiOiJnb2RkZXNzIGZpeCBjcmFuZSByaWdpZCBjYXN1YWwgd2hlYXQgc3ByYXkgY3JlYW0gY29pbiBsZWFmIGRpc3BsYXkgaHVyZGxlIGVuc3VyZSBhcm1lZCBjb3JyZWN0IGJ1bmtlciBpZGVudGlmeSBkb25hdGUgc3Bpa2UgZmFudGFzeSByZWxpZWYgZGlzYWdyZWUgZ3JhcGUgc2hvdWxkZXIifQ=="),
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMTN6OWNteHVrdzdoaHYzNXgweDU0YXZhbXJuOXA2NWF0cGcybmR4IiwiTW5lbW9uaWMiOiJtb25zdGVyIGxhbmd1YWdlIHZvY2FsIGZseSBtYXJjaCBqb2tlIG5ldXRyYWwgamVsbHkgY2h1cm4gZ29kZGVzcyB3ZWFyIGNhbG0gdmFsaWQgbm93IGF1dHVtbiBsaWFyIHdhbnQgc3Bvb24gZ2xvb20gaGlnaCBqdXN0IHNvbWVvbmUgbWFudWFsIGVuZXJneSJ9"),
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMTZjdTB4cTRyeDhlZWZqN3BrMmY5cHJmcmttejhtM3g1bWtyNjZyIiwiTW5lbW9uaWMiOiJtb3JuaW5nIG9mdGVuIGxhdGVyIGVjaG8geW91bmcgZGlsZW1tYSB1bnZlaWwgc3Vuc2V0IGdsYW5jZSBzaG9wIG9ic2VydmUgZXhwaXJlIHN0YWdlIG1lbWJlciBzaGlmdCBhY291c3RpYyBjb25maXJtIHJhZGlvIGp1ZGdlIGxhYiBtdWZmaW4gdHJpbSBnZW51aW5lIGJ1aWxkIn0="),
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMXQzcnVxZmN6aGp5cjRycDU5amNkdjZ4dmgwNnNhcDZuMzBjc2FhIiwiTW5lbW9uaWMiOiJsdW5jaCBib3Jyb3cgZGVmeSBwb29sIGxvY2FsIGdyYW50IHNoaXAgbXVzaHJvb20gYXdrd2FyZCBpbmNsdWRlIGtpZCBiZWx0IHBhaXIgaW50byBsZW5zIHRyYXZlbCByZWZsZWN0IHJvdXRlIG1pbmQgZW5hYmxlIG5lY2sga25lZSBzaXggd2lkdGgifQ=="),
-		mustParsePreGeneratedAccount("eyJBZGRyZXNzIjoiY29zbW9zMWg4MmVnOXBuN3pzM2w1bHRqdHN6ejN0MGdkNWVhejdnd3lyN2M1IiwiTW5lbW9uaWMiOiJzeXN0ZW0gdHJ5IGVhZ2VyIGNhcmQgbGVzc29uIGN1cnRhaW4gYmVjYXVzZSBmbGFtZSBpbWl0YXRlIHNpemUgc2libGluZyBsYXRlciBoaWdoIHNhbXBsZSBjbGF3IHNjYXR0ZXIgdXNlIHNlcmllcyBiYWNoZWxvciBwZXBwZXIgbmV4dCBhZ2FpbiBhbmdyeSBsb2dpYyJ9"),
+		mustParsePreGeneratedAccount("eyJhZGRyZXNzIjoicG9rdDEzc2gzem4ydjZuZmR2YzZhZHl4Y3l3ODAzcWVjcDdqMjc5bmd1cyIsIm1uZW1vbmljIjoic2lsbHkgcGVuYWx0eSBicmFzcyBhYmlsaXR5IHNheSB3YWdvbiBxdWl6IGZlYXR1cmUgdGhlbWUgZ2FtZSBwYXBlciByaXBwbGUgZmF0IHBhaXIgcGVybWl0IHJhYmJpdCB0ZXN0IHNjaWVuY2Ugc2hvZSBlbmVteSBib3ggZmxhdCBtZW1vcnkgYmV0d2VlbiJ9"),
```

Comment:
Why did we change all these

Reply:
They needed to be regenerated because they had the `cosmos` address prefix. Apparently it wasn't an issue for the contexts in which it was used but that is no longer the case.